### PR TITLE
Heart of Deimos

### PR DESCRIPTION
**Pull request by Aericio**
**Summary:**
Add bounties for Heart of Deimos, along with the bounty refresh cycle.

---
**Fixes issue (include link):**
https://github.com/WFCD/warframe-hub/issues/449

---
**Mockups, screenshots, evidence:**
image
image

Pattern mismatch is due to API https://api.warframestat.us/pc

### CODE REVIEW

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6116597&bid=20017546)

File: src/components/panels/TimePanel.vue [hunk 1]
```diff
@@ -11,9 +11,13 @@
                 night: time.state == 'night',
                 warm: time.state == 'warm',
                 cold: time.state == 'cold',
+                fass: time.active == 'fass',
+                vome: time.state == 'vome',
               }"
             >
-              <span style="text-transform: capitalize;">{{ this.$t(`time.${time.state.toLowerCase()}`) }}</span>
+              <span style="text-transform: capitalize;">{{
```
Comment (EricSihaoLin) [inline src/components/panels/TimePanel.vue:18]: Codacy found an issue: [Delete `;`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6116597)

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6116597&bid=20017546)

**Comment by TobiTenno** [comment]
time.vome looks wrong

File: src/assets/json/components.json [hunk 2]
```diff
@@ -67,14 +77,14 @@
...
-    "displayName": "Hive Mind Bounties Timer",
-    "key": "hivemind-bounties",
+    "displayName": "Entrati Bounties Timer",
+    "key": "entrati-bounties",
     "component": "BountyPanel",
     "props": {
-      "syndicate": "@hivemind",
-      "type": "HiveMind"
+      "syndicate": "@entrati",
+      "type": "Entrati"
```
Comment (TobiTenno) [inline src/assets/json/components.json:87]: ty for repurposing this 👍

File: src/assets/json/initialWorldstate.json [hunk 3]
```diff
@@ -4,9 +4,6 @@
     "news": [],
     "events": [],
     "alerts": [],
-    "nightwave": {
-      "activeChallenges": []
-    },
```
Comment (TobiTenno) [inline src/assets/json/initialWorldstate.json:9]: why moved?

File: src/assets/json/initialWorldstate.json [hunk 4]
```diff
@@ -20,20 +17,6 @@
...
-      "node": "Loading...",
-      "name": "Loading...",
-      "tile": "Loading...",
-      "planet": "Loading...",
-      "enemy": "Loading...",
-      "type": "Loading...",
-      "node_type": "Loading...",
-      "archwing": false,
-      "sharkwing": false
-    },
```
Comment (TobiTenno) [inline src/assets/json/initialWorldstate.json:36]: why moved?

File: src/assets/json/initialWorldstate.json [hunk 5]
```diff
@@ -63,48 +46,86 @@
...
-      "active": false,
-      "challenges": []
+      "expiry": "2000-01-01T01:00:00.000Z"
     },
+    "weeklyChallenges": [],
     "constructionProgress": {
       "id": "12345",
       "fomorianProgress": "0.00",
-      "razorbackProgress": "0.00",
+      "razorbackProgress": "2000-01-01T01:00:00.000Z",
```
Comment (TobiTenno) [inline src/assets/json/initialWorldstate.json]: should be a float in a string, not a timestamp

File: src/components/panels/TimePanel.vue [hunk 6]
```diff
@@ -11,9 +11,13 @@
                 night: time.state == 'night',
                 warm: time.state == 'warm',
                 cold: time.state == 'cold',
+                fass: time.active == 'fass',
+                vome: time.state == 'vome',
```
Comment (TobiTenno) [inline src/components/panels/TimePanel.vue]: should use `===`

(unsure why eslint didn't complain)

if possible, please fix the others, too


also...

```suggestion
                fass: time.active === 'fass',
                vome: time.active === 'vome',
```

**Review by TobiTenno — CHANGES_REQUESTED** [review]
apart from the cases you have in place, and the initialWorldstate changes, looks good. if you can solve my qualms, i'm all for this.


also, make sure you add `time` entries for `fass` and `vome`, as they don't currently exist, which is why we're seeing `Time.Fass`, right now.

image

**Comment by Aericio** [comment]
@TobiTenno I don't know why those were changed/moved. I didn't manually edit the worldstate file, I ran `initialWorldstateUpdater.js` to update them. Should I rollback and manually edit it, instead? It changed a lot of floats to timestamps, too...

**Comment by TobiTenno** [comment]
Probably just manually edit the values for stuff that has the wrong data type. Sorry for the confusion, you're doing what you should be.

For the time bits, there should be a time object in the langs/en.json

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=6116597&bid=20017546)

File: src/lang/de.json [hunk 7]
```diff
@@ -151,6 +151,8 @@
     "night": "Nacht",
     "warm": "Warm",
     "cold": "Kalt",
+    "vome": "vome",
+    "fass": "fass",
```
Comment (TobiTenno) [inline src/lang/de.json:155]: could you use the title case versions of these so the transformation is more of a backup?

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**